### ISSUE TITLE
Several fields can't be removed

### ISSUE THREAD

**Issue by MichaelFitzurka** [comictagger/comictagger#241]
Steps to create:
1. Open a new comic
2. Add count and/or alternate count values
3. Save Tags
4. Remove count and/or alternate count values
5. Save Tags
6. Remove Selected Item
7. Open comic again
8. Count and/or alternate count values are still there.

Perhapse related ... may or may not be a separate issue...
but the XML is off as well with a lot of empty tags.

``` xml
<?xml version='1.0' encoding='utf-8'?>
<ComicInfo>
  <Title />
  <Series>X</Series>
  <Number>1</Number>
  <Count>11</Count>
  <Volume />
  <AlternateSeries />
  <AlternateNumber />
  <StoryArc />
  <SeriesGroup />
  <AlternateCount>11</AlternateCount>
  <Summary />
  <Notes />
  <Writer />
  <Penciller />
  <Inker />
  <Colorist />
  <Letterer />
  <CoverArtist />
  <Editor />
  <Publisher />
  <Imprint />
  <Genre />
  <Web />
  <PageCount>27</PageCount>
  <Format />
  <AgeRating />
  <Characters />
  <Teams />
  <Locations />
  <ScanInformation />
  <Pages>
...
  </Pages>
</ComicInfo>
```

**Comment by MichaelFitzurka** [comment]
Problem is bigger: Count, Volume, AlternateCount, Year, Month, Day, LanguageISO, BlackAndWhite & Manga.

I put a value in every field, saved, cleared form, saved again, and this is the resulting XML.

``` xml
<?xml version='1.0' encoding='utf-8'?>
<ComicInfo>
  <Title />
  <Series />
  <Number />
  <Count>11</Count>
  <Volume>1995</Volume>
  <AlternateSeries />
  <AlternateNumber />
  <StoryArc />
  <SeriesGroup />
  <AlternateCount>11</AlternateCount>
  <Summary />
  <Notes />
  <Writer />
  <Penciller />
  <Inker />
  <Colorist />
  <Letterer />
  <CoverArtist />
  <Editor />
  <Publisher />
  <Imprint />
  <Genre />
  <Web />
  <PageCount>27</PageCount>
  <Format />
  <AgeRating />
  <Characters />
  <Teams />
  <Locations />
  <ScanInformation />
  <Pages>
... removed by me ...
  </Pages>
  <Year>1</Year>
  <Month>1</Month>
  <Day>1</Day>
  <LanguageISO>af</LanguageISO>
  <BlackAndWhite>Yes</BlackAndWhite>
  <Manga>Yes</Manga>
</ComicInfo>
```

**Comment by lordwelch** [comment]
This happened when I merged #202 I have a patch to fix this I plan to merge it tomorrow or the next day

**Comment by lordwelch** [comment]
Fixed in [1.3.2-alpha.0](https://github.com/comictagger/comictagger/releases/tag/1.3.2-alpha.0)
